spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -562,8 +562,7 @@
 
         it_behaves_like(
           'enrollment_encountering_an_exception',
-          exception_class: 'Faraday::ParsingError',
-          exception_message: /unexpected token at 'invalid'$/,
```

Comment:
Interesting...it doesn't throw a `Faraday::ParsingError` anymore. Don't think that matters to us but I'm surprised by that change in behavior